### PR TITLE
examples, integartion, vendor: use apiextentions charts

### PR DESCRIPTION
**Pull request by kopiczko**
Towards: https://github.com/giantswarm/giantswarm/issues/3314

Examples are removed because they are becoming more and more outdated.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -1,233 +0,0 @@
-# Running aws-operator Locally
```
Comment (kopiczko) [inline examples/README.md]: I removed whole examples directory. There are outdated charts, and no one cares about them.

File: Gopkg.toml [hunk 2]
```diff
@@ -41,10 +37,6 @@ required = [
   name = "github.com/cenkalti/backoff"
   version = "1.1.0"
 
-[[constraint]]
-  branch = "master"
-  name = "github.com/docker/distribution"
```
Comment (kopiczko) [inline Gopkg.toml]: `docker/distribution` and `Azure/go-autorest` weren't imported or required. I don't know why we have them here.
W/o them `dep ensure` could complete in reasonable time for me.

Comment (fgimenez) [reply Gopkg.toml]: Weird, dep would complain if they are not used. Anyway :+1:

**Comment by kopiczko** [comment]
@fgimenez I think we only need to remove `COMMON_DOMAIN_GUEST`. Otherwise all is good. Am I missing something?

**Comment by fgimenez** [comment]
I see there's a `COMMON_DOMAIN` defined, if it has been added recently (in other words, if it is not a leftover from when we didn't have separated guest and host account) and it doesn't include `k8s` I think we are good.

**Comment by kopiczko** [comment]
@fgimenez what is `COMMON_DOMAIN_GUEST` set to now?

**Comment by fgimenez** [comment]
If it hasn't changed `k8s.gauss.eu-central-1.aws.gigantic.io`

**Comment by kopiczko** [comment]
BTW we can't we keep those env vars in `.circleci/config.yml`?

**Comment by kopiczko** [comment]
I checked the circle docs and the highest level you can set env variables is `job`. With that we'd have to maintain the same var in multiple places which is dump.

**Comment by kopiczko** [comment]
Updated the COMMON_DOMAIN in circle, CI is green. Merging.

**Pull request merged**